oauth2_http/java/com/google/auth/oauth2/ImpersonatedCredentials.java

```diff
@@ -418,7 +419,7 @@ public boolean createScopedRequired() {
   @Override
   public GoogleCredentials createScoped(Collection<String> scopes) {
     return toBuilder()
-        .setScopes(new ArrayList(scopes))
+        .setScopes(new ArrayList<>(scopes))
```

Comment:
why this?

Reply:
To avoid using raw types (ArrayList vs. generic ArrayList<>)